include/avif/avif.h

```diff
@@ -1143,11 +1143,20 @@ typedef struct avifDecoder
 #if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
     // This is true when avifDecoderParse() detects a gain map.
     avifBool gainMapPresent;
-    // Do not decode the gain map image or gain map metadata if present (defaults to AVIF_TRUE).
-    // Set this to AVIF_FALSE if you DO want to decode gain map pixels and metadata.
+    // Do not decode the gain map image if present (defaults to AVIF_TRUE).
+    // Set this to AVIF_FALSE if you DO want to decode gain map pixels
```

Comment:
This is counter-intuitive but let's keep that for consistency.

Reply:
I agree, I did it this way for consistency but the more I use it, the more I think it's confusing. I changed both of them to "enable..."

Reply:
Another option but unlikely better: a big enum replacing `ignoreGainMap`, `ignoreGainMapMetadata` and `ignoreColorAndAlpha`.

Reply:
So like a bit field? Yeah i'm not fully convinced. Maybe if we'd done this from the beginning, with ignoreExif and ignoreXMP as well.

Reply:
I was thinking about an enum:

```c
enum avifParsingFilter {
  AVIF_PARSE_IMAGE_FEATURES,
  AVIF_PARSE_IMAGE_FEATURES_XMP_EXIF,
  AVIF_PARSE_IMAGE_FEATURES_GAIN_MAP_METADATA,
  AVIF_PARSE_ALL
}
enum avifDecodingFilter {
  AVIF_DECODE_COLOR_ALPHA,
  AVIF_DECODE_GAIN_MAP,
  AVIF_DECODE_ALL
}
```

but a bit field is likely more convenient, and individual fields are better than a bit field in my opinion anyway. So the current PR is fine.